tests/workbook/serialization_test_suite.cpp

```diff
@@ -822,14 +822,8 @@ class serialization_test_suite : public test_suite
 
     void test_locale_comma ()
     {
-        struct SetLocale
-        {
-            SetLocale() {xlnt_assert(setlocale(LC_ALL, "de_DE") != nullptr);} // If failed, please install de_DE locale to correctly run this test.
-            ~SetLocale() {setlocale(LC_ALL, "C");}
-        } setLocale;
-
         xlnt::workbook wb;
-        wb.load(path_helper::test_file("Issue714_local_comma.xlsx"));
+        wb.load(path_helper::test_file("Issue714_locale_comma.xlsx"));
```

Comment:
If we remove the `setlocale`, the test does not test anymore if we support comma as the decimale as the decimal separator for the current locale. The xlsx file itself does use a "." as its decimal separator.

So, we may consider renaming it to `test_parse_double`.

Reply:
If I'm not missing something, this should no longer be necessary, as the tests for the parsers already test for such cases. See my other comment above.

Reply:
I don't think the current unit tests cover all cases: see the coverage file (especially lines 580 up to 623): https://coveralls.io/jobs/156064273/source_files/35998489167